### ISSUE TITLE
`w0-signup-optimisation` GA

### ISSUE THREAD

**Issue by malomarrec** [sourcegraph/sourcegraph#22910]
`w0-signup-optimisation` has generated a significant, [positive outcome](https://docs.google.com/spreadsheets/d/10UMyz4MCDcEB7nNMxh9ZKlnCQmMsJk1_V1ufMrptGko/edit#gid=0). Let's GA the changes.

#### Scope
- [ ] GA all the changes and CTAs labeled `w0-signup-optimisation`, except the snippet CTA.
- [ ] remove the snippet CTA
- [ ] remove the feature flag
- [ ] remove the extend CTA
- [ ] Move all the `w0-signup-optimisation` issues to Done!
- [ ] #23180

**Comment by limitedmage** [comment]
>  make the search CTA dismissable and measure the event. (requires design)

@malomarrec @rrhyne Is this in scope for this cleanup? This sounds like a new feature that should be tackled separately. 

>  remove the extend CTA

Is this also in scope for this cleanup? I thought we were only removing the snippet CTA. Should we also remove the "extend" button altogether?

**Comment by rrhyne** [comment]
>make the search CTA dismissable and measure the event. (requires design)

> @malomarrec @rrhyne Is this in scope for this cleanup? This sounds like a new feature that should be tackled separately.

@limitedmage I think you are correct. I think we could release this without testing but time is a factor. Let's punt!

> remove the extend CTA

This should be removed along with the extend button we added as a test. It was experimental and didn't garner any results. I just want to make sure though, that we are only removing the button we added to a/b test, not functionality required by extensions team.
